api/controller.js

```diff
@@ -0,0 +1,37 @@
+const customers = require('../data/customers')
+
+const getindex = (req,res)=>{
+    return res.send('hi this is the homepage')
+}
+
+const getAllCustomers = (req,res)=>{
+    return res.send(customers)
+}
+
+const sanitizeEmails = (req,res)=>{
+    const specialChars = '!@#$%^&*()?/~`.,+=-_:;"' 
+    let allEmails = []
+    let cleanedEmail = []
+    customers.forEach((customer)=>{
+        allEmails.push([customer.A])
+        
+    })
+    
+    allEmails.forEach((email)=>{
+        console.log(email[0])
+        const regex = /[!#$%^&*(),?":{}|<>]/g
+        const regex2 = /(\s)/g
+        let cleaned = email[0].replace(regex, '')
+        cleanedEmail.push(cleaned)
```

Comment:
Nice, you may be able to reuse this function to sanitize the emails, in the new branch